scripts/prepare_dutchf3.py

```diff
@@ -9,49 +9,59 @@
 import logging.config
 import math
 import warnings
-from os import path
+from os import path, mkdir
 
 import fire
 import numpy as np
 from sklearn.model_selection import train_test_split
 
 
-def _get_splits_path(data_dir):
-    return path.join(data_dir, "splits")
-
-
-def _get_labels_path(data_dir):
-    return path.join(data_dir, "train", "train_labels.npy")
-
-
 def _write_split_files(splits_path, train_list, test_list, loader_type):
-    file_object = open(path.join(splits_path, loader_type + "_train_val.txt"), "w")
+    if not path.isdir(splits_path):
+        mkdir(splits_path)
+    file_object = open(path.join(splits_path,
+                       loader_type + "_train_val.txt"), "w")
     file_object.write("\n".join(train_list + test_list))
     file_object.close()
-    file_object = open(path.join(splits_path, loader_type + "_train.txt"), "w")
+    file_object = open(path.join(splits_path,
+                       loader_type + "_train.txt"), "w")
     file_object.write("\n".join(train_list))
     file_object.close()
-    file_object = open(path.join(splits_path, loader_type + "_val.txt"), "w")
+    file_object = open(path.join(splits_path,
+                       loader_type + "_val.txt"), "w")
     file_object.write("\n".join(test_list))
     file_object.close()
 
 
-def _get_aline_range(aline, per_val):
-    # Inline sections
-    test_aline = math.floor(aline * per_val / 2)
-    test_aline_range = itertools.chain(range(0, test_aline), range(aline - test_aline, aline))
-    train_aline_range = range(test_aline, aline - test_aline)
-
-    return train_aline_range, test_aline_range
+def _get_aline_range(aline, per_val, slice_steps):
+    try:
+        if slice_steps < 0:
```

Comment:
is slice_steps==0 a valid case? It should be >=1, right?

Reply:
Olga, isn't x < 0 the same as x >=1? I think the issue was with the ValueError message after this: "slice_steps cannot be a negative number". The message should be: "slice_steps cannot be zero or a negative number".